Changes

```diff
@@ -10,6 +10,22 @@ OCaml 4.09 maintenance branch
   prevent their C99-compliant snprintf conflicting with ours.
   (David Allsopp, report by Michael Soegtrop, review by Xavier Leroy)
 
+### Bug fixes:
+
+- #9326, #10125: Gc.set incorrectly handles the three `custom_*` fields,
+  causing a performance regression
+  (report by Emilio Jesús Gallego Arias, analysis and fix by Stephen Dolan,
+   code by Xavier Leroy, review by Hugo Heuzard and Gabriel Scherer)
```

Comment:
What is happening with this change entry?

Reply:
It was mis-merged in 59d268b which is at present the HEAD commit of the 4.09 branch - I double-checked that Changes was up-to-date on each branch before preparing the `alt-signal-stack-*` branches.